src/components/TopBar/Wallet/Assets/index.tsx

```diff
@@ -230,6 +242,34 @@ const AssetsIndex: FC<OHMAssetsProps> = (props: { path?: string }) => {
               );
           }
         })()}
+        {activeChain.id === 5 && (
```

Comment:
make this `activeChain.id === networks.TESTNET_GOERLI `

Reply:
✅